Android.mk

```diff
@@ -69,6 +69,13 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/host/xtest \
 		$(LOCAL_PATH)/ta/aes_perf/include \
 		$(LOCAL_PATH)/ta/socket/include
 
+ifeq ($(CFG_SECURE_DATA_PATH),y)
+CFLAGS += -DWITH_SECURE_DATA_PATH
```

Comment:
You mean change code inside TAs implementation to rely straight on the `CFG_SECURE_DATA_PATH` from the devkit config ? I will do that.